src/families/bitcoin/bridge/libcore.js

```diff
@@ -182,6 +185,10 @@ const getTransactionStatus = async (a, t) => {
     warnings.feeTooHigh = new FeeTooHigh();
   }
 
+  if (t.feesStrategy === null) {
```

Comment:
am I correct to see this is a breaking change in live-common? What is the default behavior of `ledger-live send` ?
ideally changes in live-common shouldn't break (meaning, i should be able to bump live-common on LLD/LLM without needed to change on LLM/LLD, so we need to make it a "no changes" by default, as much as possible)

Reply:
my bad I totally forgot about LLC cli, it shouldn't work anymore...
The spec wasn't really clear about this, but I think I heard during a daily that we do need user to choose a fees speed strategy.
I would say we should probably enforce a `medium` strategy when we use `ledger-live send` and add option `--feesStrategy. But I don't know I felt something is wrong with this approach

Reply:
to me we should default to feesStrategy medium yes. because actually flows like Swap don't even allow to chose the fees so you can't even configure, we have to fallback on the "safe fee"